nginx.conf

```diff
@@ -172,11 +172,22 @@
       proxy_set_header Connection "";
     }
 
-    # proxy requests to the Percona news rss feed
+    # This localtion stores static content for general pmm-server purposes.
+    # Ex.: local-rss.xml - contains Percona's news when no internet connection.
+    location /static {
```

Comment:
But please check that this does not overlap with some Grafana route

Reply:
@AlekSi, it is not.
But good point, now, I'd change to `/pmm-static`?